product_subscription/models/subscription_object.py

```diff
@@ -53,6 +53,11 @@ class SubscriptionObject(models.Model):
         required=True,
     )
     renewed = fields.Boolean(string="Renewed", compute="_compute_renewed")
+    is_trial = fields.Boolean(related="template.is_trial", read_only=True)
+    is_first_subscription = fields.Boolean(
+        string="First Subscription",
+        compute="_compute_is_first_subscription",
+    )
```

Comment:
Did you considered setting it to store = True ?

Reply:
cf other comment ^ :-)